Source/WebCore/page/LookalikeCharactersSanitizationData.h

```diff
@@ -46,6 +46,12 @@ struct LookalikeCharactersSanitizationData {
     {
     }
 
+    LookalikeCharactersSanitizationData(const String& lookalikeCharacters)
+        : domain({ })
```

Comment:
Nit - I _think_ you can also omit this initialization here, since it would use the `{ }` initializer by default.